src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class RuntimeInformation
+    {
+        private const string RuntimeName = ".NET Core";
```

Comment:
I'm not sure, primarily because I'm not sure what "FrameworkDescription" is supposed to refer to. The field is called ```RuntimeName``` but the property is called ```FrameworkDescription```. Assuming what we want is the "Runtime name" rather than the framework name, and thinking about the Nuget package side, perhaps something like this:

dnxcore50: CoreCLR
net46: CLR
netcore50: .NET Native, or MRT (I don't know what our official terminology is here for the runtime
mono/xamarin*: mono, or whatever they want

That's all very rough, I have no idea if it lines up with the intent of the API. The problem I have with trying to provide some sort of "FrameworkDescription" is that a "Framework" isn't well-defined. For example, you could plausibly say that you are running ".NET Core" assemblies on the desktop framework (we certainly support running a lot of the assemblies in this repo on the desktop framework). So returning ".NET Framework" on desktop wouldn't make sense to me (this assembly itself is part of .NET Core, isnt it?)